Aerial/Source/Models/ManifestLoader.swift

```diff
@@ -100,32 +105,31 @@ class ManifestLoader {
         do {
             let options = JSONSerialization.ReadingOptions.allowFragments
             let batches = try JSONSerialization.jsonObject(with: data,
-                                                           options: options) as! Array<NSDictionary>
+                                                           options: options)
+            
+            guard let batch = batches as? NSDictionary else {
+                NSLog("Aerial: Encountered unexpected content type for batch")
+                return
+            }
+            
+            let assets = batch["assets"] as! Array<NSDictionary>
             
-            for batch: NSDictionary in batches {
-                let assets = batch["assets"] as! Array<NSDictionary>
+            for item in assets {
+                let url = item["url-4K-SDR"] as! String
```

Comment:
What's the difference between the `url-4K-SDR` and `url-4K-HDR` videos?

Reply:
It's likely High Dynamic Range and Standard Dynamic Range. For TVs that support it, HDR has higher contrast range between the brightest and darkest parts of the image.

Reply:
Thanks @abstractpenguin!

Found another interesting read: https://github.com/JohnCoates/Aerial/issues/378#issuecomment-333063235